docs/java/release-notes.md

```diff
@@ -27,6 +27,59 @@ We highly recommend regularly updating to the latest SDK version. It will help y
 - update client libraries giving access to latest SAP services on SAP Cloud Platform and S4/HANA
 - protect yourself from bugs and breaking changes in the future
 
+
+## 3.25.0
+
+[Release Blog](https://blogs.sap.com/?p=1154386)
+
+## Compatibility Notes
+
+- To use Integration Objects, consumers must upgrade the CAP versions to avoid any issues in consumption:
```

Comment:
@jordan40 Remove this from external release notes

Reply:
removed